### ISSUE TITLE
Generator Can't Open .html Files with '<' or '>' characters inside

### ISSUE THREAD

**Issue by taigrr** [PythonNut/cryptomissive#1]
This error results from trying to encrypt a regular html file:

File "/var/message.html", line 1
    <html>
    ^
SyntaxError: invalid syntax

**Comment by PythonNut** [comment]
Are you using

``` bash
python cryptomissive_generator.py in.html out.html key
```

Or just

``` bash
python in.html out.html key
```

The latter will cause Python to interpret `in.html` as a Python file. This is obviously invalid.

The error in the docs was fixed in d94516e.

**Comment by taigrr** [comment]
Somehow using an older version. python cryptomissive_generator.py in.html
out.html key.
